src/test/java/com/puppycrawl/tools/checkstyle/DefaultLoggerTest.java

```diff
@@ -38,29 +47,33 @@
 
 public class DefaultLoggerTest {
 
-    private final Violation auditStartMessage = new Violation(1,
-            Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.auditStarted", null, null,
-            getClass(), null);
+    private static final Locale DEFAULT_LOCALE = Locale.getDefault();
 
-    private final Violation auditFinishMessage = new Violation(1,
-            Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.auditFinished", null, null,
-            getClass(), null);
+    @AfterEach
+    public void tearDown() throws Exception {
+        final Constructor<?> cons = getConstructor();
+        final Map<String, ResourceBundle> bundleCache =
+                Whitebox.getInternalState(cons.getDeclaringClass(), "BUNDLE_CACHE");
```

Comment:
Just use exact name of class,
No need to getConstuctors

Reply:
Strings will not work as 
`org.powermock.reflect.exceptions.FieldNotFoundException: No instance field named "BUNDLE_CACHE" could be found in the class hierarchy of java.lang.String. `